src/bld/java/rife/Rife2Build.java

```diff
@@ -29,6 +29,8 @@ public Rife2Build()
             .include(dependency("jakarta.servlet", "jakarta.servlet-api", version(5,0,0)))
             .include(dependency("org.eclipse.jetty", "jetty-server", version(11,0,15)))
             .include(dependency("org.eclipse.jetty", "jetty-servlet", version(11,0,15)))
+            .include(dependency("org.apache.tomcat.embed", "tomcat-embed-core", version(10, 1, 12)))
```

Comment:
Please remove spacing between commas to adhere to the version numbers on the other lines

Reply:
Please add your IDEA codestyle to git. It's obviously not the default one.

Reply:
It's that IDEA bug where it keeps resetting those indentation settings, it's really annoying.

Reply:
This one sadly can't be setup in IDEA, at least I haven't found how to do that just for the version numbers. The `throws` statement just keep resetting for me.

Reply:
Fixed.